lib/backend/constants.go

```diff
@@ -24,4 +24,8 @@ const (
 	DefaultBufferGuard datasize.ByteSize = 10 * datasize.MB
 	DefaultConcurrency int               = 10
 	DefaultListMaxKeys int               = 250
+
+	// Name and namespace of an empty blob used to check readiness.
+	ReadinessNamespace string = "readiness/blob"
+	ReadinessName      string = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
```

Comment:
nit :- `ReadineessCheckBlobHash` seems more descriptive ?

Reply:
I agree. Changed it.